binary-data-formats.qmd

```diff
@@ -0,0 +1,404 @@
+# Binary data formats {#sec-binary-data}
```

Comment:
Maybe we should freeze (https://quarto.org/docs/projects/code-execution.html#freeze) this doc so we don't need to try and rebuild on github?